### PR TITLE
[ISSUE #2795 refactor  fix init CommonUpstreamUtils NullPointer]

### LINKED ISSUE #2795: [BUG] <shenyu-admin start error.>
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

refactor convertCommonUpstreamList() ,add empty logic

### Expected Behavior

start ShenyuAdminBootstrap

### Steps To Reproduce

1.check out shenyu source code ,use master branch
2.execute script mysql.sql file
3.adjust avtive = mysql
4.run ShenyuAdminBootstrap

### Environment

```markdown
ShenYu version(s): master
```

### Debug logs

```
2022-01-19 15:30:45 [main] INFO  org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor - Inconsistent constructor declaration on bean with name 'dataPermissionController': single autowire-marked constructor flagged as optional - this constructor is effectively required since there is no default constructor to fall back to: public org.apache.shenyu.admin.controller.DataPermissionController(org.apache.shenyu.admin.service.DataPermissionService)
2022-01-19 15:30:45 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
Wed Jan 19 15:30:46 CST 2022 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
2022-01-19 15:30:46 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
2022-01-19 15:30:47 [main] WARN  org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'pluginController' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/controller/PluginController.class]: Unsatisfied dependency expressed through constructor parameter 1; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'syncDataServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SyncDataServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 2; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'selectorServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 7; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upstreamCheckService' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/UpstreamCheckService.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
2022-01-19 15:30:47 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown initiated...
Wed Jan 19 15:30:47 CST 2022 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.
2022-01-19 15:30:47 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Shutdown completed.
2022-01-19 15:30:47 [main] INFO  org.apache.catalina.core.StandardService - Stopping service [Tomcat]
2022-01-19 15:30:47 [main] INFO  org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener - 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
15:30:47,729 |-INFO in c.q.l.co.rolling.helper.RenameUtil - Renaming file [./logs/shenyu-admin-error.log] to [./logs/shenyu-admin.2022-01-18-error.0.log182446565856923.tmp]
2022-01-19 15:30:47 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'pluginController' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/controller/PluginController.class]: Unsatisfied dependency expressed through constructor parameter 1; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'syncDataServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SyncDataServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 2; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'selectorServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 7; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upstreamCheckService' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/UpstreamCheckService.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:798)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:228)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.shenyu.admin.ShenyuAdminBootstrap.main(ShenyuAdminBootstrap.java:36)
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'syncDataServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SyncDataServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 2; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'selectorServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 7; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upstreamCheckService' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/UpstreamCheckService.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:798)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:228)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:885)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:789)
	... 19 common frames omitted
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'selectorServiceImpl' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 7; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upstreamCheckService' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/UpstreamCheckService.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:798)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:228)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:885)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:789)
	... 33 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upstreamCheckService' defined in file [/Users/lishuo/dev/project/openSource/incubator-shenyu/shenyu-admin/target/classes/org/apache/shenyu/admin/service/impl/UpstreamCheckService.class]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:314)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:295)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1358)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1204)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:885)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:789)
	... 47 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.admin.service.impl.UpstreamCheckService]: Constructor threw exception; nested exception is java.lang.NullPointerException
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:213)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:117)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:310)
	... 61 common frames omitted
Caused by: java.lang.NullPointerException: null
	at org.apache.shenyu.admin.utils.CommonUpstreamUtils.convertCommonUpstreamList(CommonUpstreamUtils.java:98)
	at org.apache.shenyu.admin.service.converter.SpringCloudSelectorHandleConverter.convertUpstream(SpringCloudSelectorHandleConverter.java:38)
	at org.apache.shenyu.admin.service.impl.UpstreamCheckService.lambda$fetchUpstreamData$3(UpstreamCheckService.java:316)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.apache.shenyu.admin.service.impl.UpstreamCheckService.fetchUpstreamData(UpstreamCheckService.java:314)
	at org.apache.shenyu.admin.service.impl.UpstreamCheckService.setup(UpstreamCheckService.java:138)
	at org.apache.shenyu.admin.service.impl.UpstreamCheckService.<init>(UpstreamCheckService.java:129)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:200)
	... 63 common frames omitted
15:30:47,729 |-INFO in ch.qos.logback.core.rolling.helper.Compr
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lishuo5263**
…st  NullPointer]

// refactor fix init CommonUpstreamUtils . Fixes #2795 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java [hunk 1]
```diff
@@ -87,18 +88,22 @@ public static GrpcUpstream buildDefaultGrpcUpstream(final String host, final Int
...
+
     /**
      * Convert common upstream list list.
      *
      * @param upstreamList the upstream list
      * @return the list
      */
     public static List<CommonUpstream> convertCommonUpstreamList(final List<? extends CommonUpstream> upstreamList) {
-        return upstreamList.stream().map(upstream -> new CommonUpstream(upstream.getProtocol(), upstream.getUpstreamHost(), upstream.getUpstreamUrl())).collect(Collectors.toList());
+        return Optional.ofNullable(upstreamList)
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:99]: when upstreamList can be null？

Comment (lishuo5263) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:99]: * When shenyu-admin first starting ,
* In UpstreamCheckService.java line 316, converUpstream has error.

**Comment by yu199195** [comment]
@lishuo5263  

1. selector getHandle() is  not null, why? 
2. pls checked ci

**Comment by lishuo5263** [comment]
> @lishuo5263
> 
> 1. selector getHandle() is  not null, why?
> 2. pls checked ci

### 1.First of all,  I activied springcloud plugin,status is open.
### 2.I got selectorDO.getHandle() result is -> {"serviceId":"springCloud-test","gray":false}
### 3.The next invoke SpringCloudSelectorHandleConverter.convertUpstream()
### 4.pls look the png, here is Nullpointer reason
### 5.The next move is commit fix lambda logic
image

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java [hunk 2]
```diff
@@ -42,19 +43,19 @@
     public static DivideUpstream buildDefaultDivideUpstream(final String host, final Integer port) {
         return DivideUpstream.builder().upstreamHost("localhost").protocol("http://").upstreamUrl(buildUrl(host, port)).weight(50).warmup(10).timestamp(System.currentTimeMillis()).build();
     }
-    
+
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:46]: why not modify this

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java [hunk 3]
```diff
@@ -42,19 +43,19 @@
...
     /**
      * Build divide upstream divide upstream.
      *
      * @param protocol the protocol
-     * @param host the host
-     * @param port the port
+     * @param host     the host
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:51]: why not modify this

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java [hunk 4]
```diff
@@ -65,7 +66,7 @@ public static DivideUpstream buildDivideUpstream(final String protocol, final St
     public static DubboUpstream buildDefaultDubboUpstream(final String host, final Integer port) {
         return DubboUpstream.builder().upstreamHost("localhost").protocol("dubbo://").upstreamUrl(buildUrl(host, port)).weight(50).warmup(10).timestamp(System.currentTimeMillis()).build();
     }
-    
+
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: why not modify this

Comment (yu199195) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: you can format this code，

Comment (lishuo5263) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: E, The Idea shortcuts default formated . please Ingore those default formated, merge  convertCommonUpstreamList()  method That's enough .

Comment (lishuo5263) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: E, The Idea shortcuts default formated . please Ingore those default formated, merge  convertCommonUpstreamList()  method That's enough .

Comment (yu199195) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: hi， pls only modify convertCommonUpstreamList， not modify other~

Comment (lishuo5263) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java:69]: Ok, I revert  unnecessary formated, please check.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**